### PR TITLE
Update api.rst

### PR DESCRIPTION
**Pull request by ramiroluz**

### CODE REVIEW

File: mastering-plone/api.rst [hunk 1]
```diff
@@ -231,7 +231,7 @@ Exercise
 
     * Since calling view is a GET and not a POST we need ``alsoProvides(self.request, IDisableCSRFProtection)`` to allow write-on-read without Plone complaining. Alternatively we could create a simple form and create the content on submit.
     * https://docs.plone.org/develop/plone.api/docs/content.html#transition. ``transition`` has two modes of operation: The documented one is ``api.content.transition(obj=foo, transition='bar')``. That mode tries to execute that specific tranistion. But sometimes it is better to use `to_state` which tries to to find a way to get from the current state to the target-state. See https://docs.plone.org/develop/plone.api/docs/api/content.html#plone.api.content.transition for the docstring.
-    * To use methods like ``cropText`` from another view you sue the method  already discussed in
+    * To use methods like ``cropText`` from another view you use the method  already discussed in
     * Here the joke is added as the description. To add it as the text you'd need to create a instance of ``RichTextValue`` and set that as a attribute:
```
Comment (stevepiercy) [inline mastering-plone/api.rst]: While you're in this file...

- s/text you'd/text, you
- s/create a instance/create an instance
- s/a attribute/an attribute

File: mastering-plone/api.rst [hunk 2]
```diff
@@ -231,7 +231,7 @@ Exercise
 
     * Since calling view is a GET and not a POST we need ``alsoProvides(self.request, IDisableCSRFProtection)`` to allow write-on-read without Plone complaining. Alternatively we could create a simple form and create the content on submit.
     * https://docs.plone.org/develop/plone.api/docs/content.html#transition. ``transition`` has two modes of operation: The documented one is ``api.content.transition(obj=foo, transition='bar')``. That mode tries to execute that specific tranistion. But sometimes it is better to use `to_state` which tries to to find a way to get from the current state to the target-state. See https://docs.plone.org/develop/plone.api/docs/api/content.html#plone.api.content.transition for the docstring.
-    * To use methods like ``cropText`` from another view you sue the method  already discussed in
+    * To use methods like ``cropText`` from another view you use the method  already discussed in
```
Comment (stevepiercy) [inline mastering-plone/api.rst]: - s/view you use/view, you can use
- Remove extra space between "method" and "already"

**Comment by gforcada** [comment]
I could try, but maybe @ramiroluz wants to pick that up, he is the PR author, I just reviewed and merged :smile: or, you could do that yourself as well :wink:

**Comment by ramiroluz** [comment]
I found a small typo in the chapter Dexterity Types II: Growing Up. I can do both changes and create a new pull request.

**Comment by ramiroluz** [comment]
Referencing #381.

**Pull request merged**